subcommands/devices/config_updates.go

```diff
@@ -57,17 +63,21 @@ func doConfigUpdates(cmd *cobra.Command, args []string) {
 	subcommands.DieNotNil(err, "Failed to fetch a device:")
 
 	subcommands.SetUpdatesConfig(&subcommands.SetUpdatesConfigOptions{
-		UpdateApps: updateApps,
-		UpdateTag:  updateTag,
-		IsDryRun:   isDryRun,
-		IsForced:   isForced,
-		Device:     device,
+		UpdateApps:      updateApps,
+		UpdateAppsGiven: updateAppsGiven,
+		InheritApps:     inheritApps,
+		UpdateTag:       updateTag,
+		UpdateTagGiven:  updateTagGiven,
+		InheritTag:      inheritTag,
+		IsDryRun:        isDryRun,
+		IsForced:        isForced,
```

Comment:
This boilerplate (as well as option parsing above) is _exactly the same_ in all 3 config handlers.

I think, as the amount of options (and underlying logic) increased quite some, it makes sence to make a common function like `ParseConfigOptions(cmd *cobra.Command) SetUpdatesConfigOptions`, which parses these _known_ options in a uniform way.

Reply:
Gotcha, I will try and implement that.